Matchismo/Matchismo/Model/SetCard.h

```diff
@@ -0,0 +1,28 @@
+//
+//  SetCard.h
+//  Matchismo
+//
+//  Created by Otto Olkkonen on 15/06/2022.
+//
+
+#import "Card.h"
+
+#import <UIKit/UIKit.h>
+
+@interface SetCard : Card
+
+typedef enum {
+  setCardShadingOpen,
+  setCardShadingSolid,
+  setCardShadingStriped,
+  setCardShadingCount
+} SetCardShading;
+
+@property (nonatomic) NSUInteger noOfShapes;
```

Comment:
Preferable not to use abbreviations like this in property names, `numberOfShapes` is better.